dietpi/func/dietpi-wifidb

```diff
@@ -259,24 +253,22 @@ _EOF_
 		)
 
 		G_WHIP_DEFAULT_ITEM="${aWIFI_KEYMGR[$WIFI_SSID_INDEX]}"
-		G_WHIP_MENU 'Please select a WiFi encryption mode.\n\nNB: If unsure, its most likely WPA-PSK.'
-		if (( $? == 0 )); then
+		G_WHIP_MENU 'Please select a WiFi encryption mode.\n\nNB: If unsure, its most likely WPA-PSK.'; local EXIT_CODE=$?
```

Comment:
if local should be lowercase.

Init following at start of function?
```
local return_code=0
```

Then use ```return_code``` here? and ```$return return_code```?

Reply:
@Fourdee 
Ah yes this can be local.